v8engine/src/main/java/cm/aptoide/pt/v8engine/analytics/Analytics.java

```diff
@@ -233,6 +243,8 @@ public static void onCreate(android.app.Application application) {
       private static void setupDimensions(android.app.Application application) {
         if (!checkForUTMFileInMetaINF(application)) {
```

Comment:
this logic should be separated. One method to update utms, other to check if it has utms